### ISSUE TITLE
Adding import during content assist broken when augmenting exsiting import

### ISSUE THREAD

**Issue by NumberFourAG** [NumberFour/n4js#92]
IDEBUG-764
 Having a module "M" exporting two classes A and B, then content assist fails in the following situation:

```
import {A} from "M"

var A a;
var B  // <-- place cursor here
```

If you place the cursor behind the B in the last line, press CMD+Space and select M.B, then the content assist will try to augment the existing import for A. This breaks and shows an exception on stdout.

Problem is probably, that this code uses the standard Xtext serializer which does not work at the moment. This should be investigated by Moritz as part of IDE-2042.

Note that this only happens if an import already exists for M, because otherwise other code will be executed that does not use the serializer and that seems to work.

**Comment by torstenkraemer** [comment]
checked the behaviour with the ES4-type annotations, the Example above works as expected

```
import {A} from "M";
var a: A;
var b: B; 
```

leads to:

```
import {A} from "M";
import { B } from "M"
var a: A;
var b: B; 
```

which is fine.

**Comment by torstenkraemer** [comment]
Detected another issue in cases where whitespace is in front of the first import statement:

```
    // 
import {A} from "M";

var a: A;
var b: B; 
```

on pressing CMD+1 after "B;" leads to 

```
    // 
import {A} from "M";

var a
import { B } from "M": A;
var b: B; 
```

So apparently the amount of white space before the first import statement seems to shift the insertion point of the quick-fixed import line and messes up the content of the file.

**Comment by torstenkraemer** [comment]
Merged by 720be2f72f31e91798cd76df62b7bdb98d63966b

**Comment by yoosiba** [comment]
looks related : #177
